app/config/locale/translations/de.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "de",
     "settings.direction": "ltr",
     "emails.sender": "%s Team",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Kontoverifizierung",
+    "emails.verification.hello": "Hey {{name}}",
+    "emails.verification.body": "Folge diesem Link, um deine E-Mail-Adresse zu bestätigen.",
+    "emails.verification.footer": "Solltest du keine Verifizierung dieser E-Mail-Adresse angefordert haben, kannst du diese Nachricht ignorieren.",
+    "emails.verification.thanks": "Danke",
+    "emails.verification.signature": "{{project}}-Team",
+    "emails.magicSession.subject": "Login",
+    "emails.magicSession.hello": "Hey,",
+    "emails.magicSession.body": "Folge diesem Link, um dich einzuloggen.",
+    "emails.magicSession.footer": "Solltest du keinen Login für diese E-Mail-Adresse angefordert haben, kannst du diese Nachricht ignorieren.",
+    "emails.magicSession.thanks": "Danke",
+    "emails.magicSession.signature": "{{project}}-Team",
+    "emails.recovery.subject": "Kennwort zurücksetzen",
+    "emails.recovery.hello": "Hallo {{name}}",
+    "emails.recovery.body": "Folge diesem Link, um dein {{project}}-Kennwort zurückzusetzen.",
+    "emails.recovery.footer": "Solltest du keine Kennwort-Zurücksetzung angefordert haben, kannst du diese Nachricht ignorieren.",
+    "emails.recovery.thanks": "Danke",
+    "emails.recovery.signature": "{{project}}-Team",
+    "emails.invitation.subject": "Einladung zum %s-Team auf %s",
+    "emails.invitation.hello": "Hello",
+    "emails.invitation.body": "Du erhälst diese E-Mail, weil {{owner}} dich in das Team {{team}} auf {{project}} eingeladen hat.",
+    "emails.invitation.footer": "Wenn du nicht interessiert bist, kannst Sie diese Nachricht ignorieren.",
```

Comment:
Change `Sie` -> `du`. Otherwise the sentence wouldn't make much sense. Regards.

Reply:
Whoops. That happens, if you are unsure, what to use 😂